core/pkg/resultshandling/printer/v2/prettyprinter/tableprinter/imageprinter/utils.go

```diff
@@ -77,19 +77,19 @@ func getImageScanningColumnsAlignments() []int {
 	return []int{tablewriter.ALIGN_CENTER, tablewriter.ALIGN_LEFT, tablewriter.ALIGN_LEFT, tablewriter.ALIGN_LEFT, tablewriter.ALIGN_LEFT}
 }
 
-func getColor(severity string) color.Attribute {
+func getColor(severity string) (func(...string) string) {
```

Comment:
@Daniel-GrunbergerCA Why is this function implemented here and in the [utils](core/pkg/resultshandling/printer/v2/prettyprinter/tableprinter/utils/utils.go) file?